app/jobs/reports/identity_verification_report.rb

```diff
@@ -16,7 +16,7 @@ def perform(report_date)
 
       save_report(REPORT_NAME, csv, extension: 'csv')
 
-      if emails.empty?
+      if emails.compact.empty?
```

Comment:
1. I might suggest moving this into the `def emails` method itself, since I doubt we'd ever want to deal with a `[nil]` array from that method
2. Could we add some regression test coverage for this?